.github/component_owners.yml

```diff
@@ -79,6 +79,8 @@ components:
   src/OpenTelemetry.ResourceDetectors.ProcessRuntime/:
     - Kielek
     - lachmatt
+  src/OpenTelemetry.Resources.Gcp/:
+    - matt-hensley
```

Comment:
It is good practice to have at least 2 component owners. Do you have any other OTel contributor in your organization who can work as the second owner?